custom/icds_reports/sqldata/exports/aww_activity_report.py

```diff
@@ -54,12 +54,18 @@ def _format_date(data):
                 row['block_name'],
                 row['supervisor_name'],
                 row['awc_name'],
-                row['awc_site_code'],
-                _format_date(_format_infrastructure_data(row['first_submission'])),
-                _format_date(_format_infrastructure_data(row['last_submission'])),
-                _format_infrastructure_data(row['no_of_days_since_start']),
-                _format_infrastructure_data(row['no_of_days_inactive'])
+                row['awc_site_code']
             ]
+            if row['first_submission'] is None or row['first_submission'] == '':
```

Comment:
nit: could be replaced with `if not row['first_submission']`

Reply:
updated!